yarn-project/end-to-end/src/fixtures/snapshot_manager.ts

```diff
@@ -255,11 +255,18 @@ async function setupFromFresh(
 
   // Deploy our L1 contracts.
   logger.verbose('Deploying L1 contracts...');
-  const hdAccount = mnemonicToAccount(MNEMONIC);
-  const privKeyRaw = hdAccount.getHdKey().privateKey;
-  const publisherPrivKey = privKeyRaw === null ? null : Buffer.from(privKeyRaw);
-  const deployL1ContractsValues = await setupL1Contracts(aztecNodeConfig.l1RpcUrl, hdAccount, logger);
+  const hdAccount = mnemonicToAccount(MNEMONIC, { accountIndex: 0 });
```

Comment:
Is there a reason to not use the `getPrivateKeyFromIndex` that you added in `utils.ts` for getting the keys and then the hdAccount for `setupL1Contracts` separately?

Reply:
Ah i think this was when debugging i reverted using that helper and forgot to put it back

Reply:
ah i see, the deploy contracts beloy uses the hd account object, so the helper cant be used here